### PR TITLE
feat: add the bussiness clocks table for variety needs

### PR DESCRIPTION
**Pull request by ai-chen2050**

### CODE REVIEW

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.15
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "bussiness_clock")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
```
Comment (bufrr) [inline zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: Can ID be a negative number?

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    pub category: String,
+    pub address: String,
+    pub clock: Json,
+    pub clock_hash: String,
+    pub tag: String,
+    pub event_count: i32,
```
Comment (bufrr) [inline zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: Can count be a negative number?

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    pub message_id: String,
```
Comment (bufrr) [inline zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: Consider Vec<u8>

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "bussiness_clock")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub category: String,
+    pub address: String,
+    pub clock: Json,
+    pub clock_hash: String,
```
Comment (bufrr) [inline zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: should be `Vec<u8>`

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "bussiness_clock")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub category: String,
+    pub address: String,
```
Comment (bufrr) [inline zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: should be `Vec<u8>`

**Review by bufrr — CHANGES_REQUESTED** [review]

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs (hunk 1 above)
Comment (ai-chen2050) [reply zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: All the entities are generated by the see-orm client.  And Postgres db has no uint integer types.

File: zchronod/db_sql/src/pg/entities/bussiness_clock.rs (hunk 3 above)
Comment (ai-chen2050) [reply zchronod/db_sql/src/pg/entities/bussiness_clock.rs]: hex string for querying convenient

**Pull request merged**